add 'pixels' units for mouseevent coordinates

Simon-D-R opened the pull request:
<!-- 🙌 Thanks for contributing to MDN Web Docs. Adding details below will help us to merge your PR faster. -->

### Description

Added an explicit 'pixels' unit to the various coordinates associated with MouseEvent.
This was already the case for layerX, layerY, pageX and pageY.
Also made pageY more similar to pageX.

### Motivation

There might be some confusion as to what units these coordinates are given in. I personally went to look outside of MDN to make sure it was indeed in pixels.

### Additional details

I chose to adopt the way layerX, layerY, pageX and pageY specified the unit, by putting it in the __value__ section.

Along the way, I noticed the pageX was more extensive than pageY, so I duplicated (mutatis mutandis) the info.

### Related issues and pull requests

<!-- 🔨 If this fully resolves a GitHub issue, use "Fixes #123" -->
<!-- 👉 Highlight related pull requests using "Relates to #123" -->
<!-- ❗ If another pull request should be merged first, use "**Depends on:** #123" -->


<!-- 👷‍♀️ After submitting, go to the "Checks" tab of your PR for the build status -->

Simon-D-R committed:
add 'pixels' units for mouseevent coordinates

Mouvedia commented on files/en-us/web/api/mouseevent/pagey/index.md:
```diff
@@ -8,12 +8,26 @@ browser-compat: api.MouseEvent.pageY
-The **`pageY`** read-only property of the {{domxref("MouseEvent")}} interface returns the Y (vertical) coordinate in pixels of the event relative to the whole document.
-This property takes into account any vertical scrolling of the page.
+The **`pageY`** read-only property of the {{domxref("MouseEvent")}} interface returns the Y (vertical) coordinate (in pixels) at which the mouse was clicked, relative to the left edge of the entire document.
+This includes any portion of the document not currently visible.
```
> to the _left_ edge

Is it really the left edge?

SimonDeRidder replied:
You're right, it should be "top edge" here. Thanks for the review!

Simon-D-R committed:
Fix "left edge" got pageY

Fixed a reference to the "left edge" of the document where it should be the "top edge" for the pageY property

Josh-Cena committed:
Update index.md

Josh-Cena reviewed (approved):
Thanks a lot!

It has been waiting for a while so I'll just merge it, but FYI, I've reverted your changes to `pageY`. As much as I love consistency, I don't think most of the existing text in `pageX` is necessary because they are talking about how the spec _used_ to look, which is probably a long distant memory. Having a single source of this kind of historical reference makes it easier to update in the future so you don't risk out of sync content.

Josh-Cena committed:
Update index.md
